shared/lib-metrics/src/test/java/gov/va/vro/metricslogging/MetricLoggerServiceTest.java

```diff
@@ -136,27 +135,28 @@ void testGetTimestamp() {
     assertTrue(timestampToTest < (timestamp + 10));
   }
 
-  @Test
-  void testCreateDistributionPointsPayload() {
-    double timestamp = MetricLoggerService.getTimestamp();
-
-    DistributionPointsPayload dpl =
-        mls.createDistributionPointsPayload(
-            MetricLoggerService.METRIC.REQUEST_DURATION,
-            timestamp,
-            1523,
-            new String[] {"food:pizza"});
-    Assertions.assertEquals(1, dpl.getSeries().size());
-    Assertions.assertEquals(
-        String.format("%s.request_duration", METRICS_PREFIX), dpl.getSeries().get(0).getMetric());
-    Assertions.assertEquals(
-        new DistributionPointItem(timestamp), dpl.getSeries().get(0).getPoints().get(0).get(0));
-    Assertions.assertEquals(
-        new DistributionPointItem(List.of(1523.0)),
-        dpl.getSeries().get(0).getPoints().get(0).get(1));
-    Assertions.assertTrue(
-        Objects.requireNonNull(dpl.getSeries().get(0).getTags()).contains("food:pizza"));
-  }
+  // @Test
```

Comment:
Looking good! Let's add a code comment detailing why this test is commented out!

Reply:
I added a comment to this